src/app/components/InputPageButton/InputPageButton.tsx

```diff
@@ -15,6 +15,16 @@ export default function InputPageButton({
 }
 
 const Button = styled.button`
-  padding: 0.4em 1em;
-  justify-self: center;
+  display: inline-block;
+  height: 2.5em;
+  padding: 0.5em 1.3em;
+  background-color: var(--color-dark-green);
+  color: var(--color-background-light);
+  border: none;
+  border-radius: 0.8em;
+  margin-left: 1.9em;
```

Comment:
Would it make sense to use a padding of the parent container instead of a margin for the children? Sometimes it's not possible, but maybe you can check that out :)